### ISSUE TITLE
Feedback image support needs documenting & adding to schema files

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-core#407]
### Subject of the issue
Feedback was expanded to support images and set alignment. See PR https://github.com/adaptlearning/adapt-contrib-core/pull/223 

- No documentation
- New properties missing from schemas.
